packages/uniforms-antd/src/BoolField.js

```diff
@@ -1,26 +1,38 @@
 import Icon           from 'antd/lib/icon';
 import React          from 'react';
 import Switch         from 'antd/lib/switch';
+import Checkbox       from 'antd/lib/checkbox';
 import connectField   from 'uniforms/connectField';
 import filterDOMProps from 'uniforms/filterDOMProps';
 
 import wrapField from './wrapField';
 
 const Bool = props =>
     wrapField(props, (
-        <Switch
-            checked={props.value}
-            disabled={props.disabled}
-            id={props.id}
-            name={props.name}
-            onChange={() => props.onChange(!props.value)}
-            ref={props.inputRef}
-            {...filterDOMProps(props)}
-        />
+        props.checkbox ?
+            <Checkbox
+                checked={props.value}
+                disabled={props.disabled}
+                id={props.id}
+                name={props.name}
+                onChange={() => props.onChange(!props.value)}
+                ref={props.inputRef}
+                {...filterDOMProps(props)}
+            /> :
+            <Switch
+                checked={props.value}
+                disabled={props.disabled}
+                id={props.id}
+                name={props.name}
+                onChange={() => props.onChange(!props.value)}
+                ref={props.inputRef}
+                {...filterDOMProps(props)}
+            />
```

Comment:
How about `React.createElement(props.checkbox ? Checkbox : Switch, {...})`? It'll reduce this duplicated code.